.circleci/config.yml

```
@@ -100,7 +100,7 @@ jobs:
 executors:
   xcode-11:
     macos:
-      xcode: "11.3.1"
+      xcode: "11.4"
```

Not sure what happens if it's missing the ".0": https://circleci.com/docs/2.0/testing-ios/#supported-xcode-versions

```suggestion
      xcode: "11.4.0"
```